src/components/Dashboard/TodoList.vue

```diff
@@ -278,6 +251,9 @@
                     <template v-else-if="isTypeRegistration(item) && item.isEmptyFiling">
                       <span>Register using this NR</span>
                     </template>
+                    <!-- <template v-else-if="item.isAlteringToBen">
+                      <span>Alter Now</span>
+                    </template> -->
```

Comment:
I'm waiting to hear back from Mihai whether to have this or to use Resume.

Reply:
Screenshot:

![image](https://user-images.githubusercontent.com/10002889/207436076-80abbb80-3be9-4190-8d71-d943cde128a0.png)

Reply:
Decision was to show Resume action and no blurb (same as other draft filings).